flatgfa/src/cli/cmds.rs

```diff
@@ -299,3 +301,50 @@ pub fn gaf_lookup(gfa: &flatgfa::FlatGFA, args: GAFLookup) {
         }
     }
 }
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-import")]
+pub struct SeqImport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_import(args: SeqImport) {
+    let vec = packedseq::import(&args.filename);
+    let _ = std::fs::remove_file(&args.filename);
+    let store = PackedSeqStore::create(vec);
+    let view = store.as_ref();
+    print!("{}", view);
+}
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-export")]
+pub struct SeqExport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_export(args: SeqExport) {
+    let mut input = String::new();
+    std::io::stdin()
+        .read_to_string(&mut input)
+        .expect("Stdin read failure");
+
+    let vec: Vec<packedseq::Nucleotide> = input
+        .chars()
+        .filter(|c| !c.is_whitespace())
+        .map(|c| match c {
+            'A' => packedseq::Nucleotide::A,
+            'C' => packedseq::Nucleotide::C,
+            'T' => packedseq::Nucleotide::T,
+            'G' => packedseq::Nucleotide::G,
+            _ => panic!("Invalid character in fle: {}", c),
+        })
```

Comment:
What do you think about providing this functionality in an `impl From<char> for Nucleotide`? Seems like it would fit nicely with all the other various `From` implementations that `Nucleotide` already has.